x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/cohere/CohereServiceSettings.java

```diff
@@ -48,49 +53,59 @@ public static CohereServiceSettings fromMap(Map<String, Object> map) {
         URI uri = convertToUri(url, URL, ModelConfigurations.SERVICE_SETTINGS, validationException);
         String model = extractOptionalString(map, MODEL, ModelConfigurations.SERVICE_SETTINGS, validationException);
 
+        String modelId = extractOptionalString(map, MODEL_ID, ModelConfigurations.SERVICE_SETTINGS, validationException);
```

Comment:
I guess we haven't technically released cohere yet, but since we just merged the serverless PR maybe it's safer to support both `model` and `model_id` like I have here 🤷‍♂️